pkg/tests/apis/provisioning/helper_test.go

```diff
@@ -45,7 +45,6 @@ type provisioningTestHelper struct {
 
 	Repositories *apis.K8sResourceClient
 	Jobs         *apis.K8sResourceClient
-	HistoricJobs *apis.K8sResourceClient
```

Comment:
removed direct access to this storage in tests -- we now use the sub-resource to check when jobs are done